src/backend/distributed/citus--7.3-2--7.3-3.sql

```diff
@@ -0,0 +1,75 @@
+/* citus--7.3-2--7.3-3 */
+
+/*****************************************************************************
+ * Citus json aggregate helpers
+ *****************************************************************************/
+
+CREATE FUNCTION pg_catalog.citus_jsonb_concatenate(state jsonb, val jsonb)
+	RETURNS jsonb
+	LANGUAGE SQL
+AS $function$
+	SELECT CASE 
+		WHEN val IS NULL THEN state
+		WHEN jsonb_typeof(state) = 'null' THEN val
+		ELSE state || val
+	END;
+$function$;
+
+CREATE FUNCTION pg_catalog.citus_jsonb_concatenate_final(state jsonb)
+	RETURNS jsonb
+	LANGUAGE SQL
+AS $function$
+	SELECT CASE WHEN jsonb_typeof(state) = 'null' THEN NULL ELSE state END;
+$function$;
+
+CREATE FUNCTION pg_catalog.citus_json_concatenate(state json, val json)
+	RETURNS json
+	LANGUAGE SQL
+AS $function$
+	SELECT CASE 
+		WHEN val IS NULL THEN state
+		WHEN json_typeof(state) = 'null' THEN val
+		WHEN json_typeof(state) = 'object' THEN
+ 			(SELECT json_object_agg(key, value) FROM (
+		 		SELECT * FROM json_each(state)
+		 		UNION ALL
+		 		SELECT * FROM json_each(val)
+	 		) t)
+		ELSE 
+	 		(SELECT json_agg(a) FROM (
+	 			SELECT json_array_elements(state) AS a
+		 		UNION ALL
+		 		SELECT json_array_elements(val) AS a
+	 		) t)
+	END;
+$function$;
+
+CREATE FUNCTION pg_catalog.citus_json_concatenate_final(state json)
+	RETURNS json
+	LANGUAGE SQL
+AS $function$
+	SELECT CASE WHEN json_typeof(state) = 'null' THEN NULL ELSE state END;
+$function$;
+
+
+/*****************************************************************************
+ * Citus json aggregates
+ *****************************************************************************/
+
+CREATE AGGREGATE pg_catalog.jsonb_cat_agg(jsonb) (
```

Comment:
It looks like we've broken `PARALLEL ` fields of these aggregates. Before citus created:
```SQL
postgres=# SELECT p1.oid, p1.proname
postgres-# FROM pg_proc AS p1
postgres-# WHERE provolatile = 'i' AND proparallel = 'u';
 oid | proname 
-----+---------
(0 rows)

postgres=# create extension citus ;
CREATE EXTENSION
postgres=# SELECT p1.oid, p1.proname
postgres-# FROM pg_proc AS p1
postgres-# WHERE provolatile = 'i' AND proparallel = 'u';
  oid  |    proname    
-------+---------------
 16492 | array_cat_agg
 16773 | jsonb_cat_agg
 16774 | json_cat_agg
(3 rows)
```

I hit this while running Postgresql's regression test `check-installed` when Citus is created.

Is that intended?